src/main/java/org/folio/edge/sip2/repositories/CirculationRepository.java

```diff
@@ -614,7 +614,8 @@ public Future<RenewResponse> performRenewCommand(Renew renew,
   }
   
   RenewAllResponseBuilder doRenewals(JsonObject jo, RenewAllResponseBuilder builder) {
-    log.debug(jo.toString());
+    
+    log.debug(jo != null ? jo.encode() : "Null json object passed to doRenewals");
```

Comment:
@kurtnordstrom let's use something like this to shut up sonar:

log.debug("doRenawals: {} ", jo != null ? jo.encode() : null);